testgrid.tahoe-lafs.org/hardware-configuration.nix

```diff
@@ -0,0 +1,23 @@
+# Do not modify this file!  It was generated by ‘nixos-generate-config’
+# and may be overwritten by future invocations.  Please make changes
+# to /etc/nixos/configuration.nix instead.
```

Comment:
Pardon my ignorance, but: why should be a generated file committed to version control?  Also, what do we do if/when more hardware is added to the grid?